backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -137,23 +78,41 @@ private MimeMessage createEmail(String subject, String bodyText) throws Messagin
 
     email.setFrom(new InternetAddress(APPLICATION_EMAIL));
     email.addRecipient(RecipientType.TO, new InternetAddress(recipientEmail));
-    email.setSubject(subject);
+    email.setSubject(subjectText);
     email.setText(bodyText);
     return email;
   }
 
   /** Function that access its api and using it sends an email */
-  //   TODO(#35): Create a dummy email for ad lib itself to send emails.
   //   TODO(#36): Replace body to send real link to user instead of generic.
-  public void notifyUser() throws MessagingException, IOException, GeneralSecurityException {
-    NetHttpTransport HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
+  public void sendMatchEmail(String firstMatchRecipientName, String firstMatchRecipientEmail)
```

Comment:
recipientName, recipientEmail now that "first" doesn't differentiate two recipients.

Reply:
Ack